stl/inc/atomic

```diff
@@ -427,6 +457,45 @@ struct _Atomic_storage {
         return _Result;
     }
 
+#if _HAS_CXX20
+    void wait(_Ty _Expected, const memory_order _Order = memory_order_seq_cst) const noexcept {
+        const auto _Storage_ptr = _STD addressof(_Storage);
+        auto _Expected_ptr      = _STD addressof(_Expected);
+
+        for (;;) {
+            _Ty _Observed      = load(_Order);
+            auto _Observed_ptr = _STD addressof(_Observed);

```

Comment:
And it appears that both `_Observed` and `_Observed_ptr` aren't modified, only read:
```suggestion
            const _Ty _Observed      = load(_Order);
            const auto _Observed_ptr = _STD addressof(_Observed);
```